gc/base/standard/ObjectScanner.hpp

```diff
@@ -190,6 +191,10 @@ class GC_ObjectScanner : public MM_BaseVirtual
 	MMINLINE static bool isIndexableObject(uintptr_t flags) { return (0 != (indexableObject & flags)); }
 
 	MMINLINE bool isIndexableObject() { return (0 != (indexableObject & _flags)); }
+
+	MMINLINE static bool isIndexableObjectNoSplit(uintptr_t flags) { return (0 != (indexableObjectNoSplit & flags)); }
```

Comment:
Not presently used in OMR (no split). GCConfigTest does not implement indexable objects.